apps/audits/serializers.py

```diff
@@ -38,10 +38,12 @@ class JobsAuditSerializer(JobSerializer):
     material = serializers.ReadOnlyField(label=_("Command"))
     summary = serializers.ReadOnlyField(label=_("Summary"))
     crontab = serializers.ReadOnlyField(label=_("Execution cycle"))
+    is_periodic_display = serializers.BooleanField(read_only=True, source='is_periodic')
 
     class Meta(JobSerializer.Meta):
         read_only_fields = [
-            "id", 'name', 'args', 'material', 'type', 'crontab', 'interval', 'date_last_run', 'summary', 'created_by'
+            "id", 'name', 'args', 'material', 'type', 'crontab', 'interval', 'date_last_run', 'summary', 'created_by',
+            'is_periodic_display'
         ]
         fields = read_only_fields + ['is_periodic']
 
```

Comment:
In the code provided, there is an inconsistency between reading-only fields being set to `read_only` in `Meta`, but not listed explicitly in the dictionary of read-only fields as shown at line [36]. Additionally, it's unclear what needs to be modified here based on the information given. The current implementation appears correct according to the latest knowledge cut-off (September 1, 2021). 

To check if there are any changes needed after September 1st, please review recent updates specifically related to this feature that were made since I last checked.